src/vs/workbench/electron-browser/workbench.ts

```diff
@@ -74,6 +74,11 @@ import {IEnvironmentService} from 'vs/platform/environment/common/environment';
 import {ITelemetryService} from 'vs/platform/telemetry/common/telemetry';
 import * as watermark from 'vs/workbench/parts/watermark/watermark';
 
+// These three are used for the macOS inline toolbars option, and can be removed once default
+import { IConfigurationService } from 'vs/platform/configuration/common/configuration';
+import { ConfigurationService } from 'vs/platform/configuration/node/configurationService';
+import { IWindowSettings } from 'vs/code/electron-main/window';
```

Comment:
this is a layer break: we cannot depend from a VSCode window into the main process, please see https://github.com/Microsoft/vscode/wiki/Code-Organization